spec/models/course/assessment/question/multiple_response_spec.rb

```diff
@@ -48,5 +48,26 @@
         end
       end
     end
+
+    describe '#type' do
+      context 'when question has more than one correct option' do
+        subject { build(:course_assessment_question_multiple_response) }
+        let!(:another_correct_option) do
+          create(:course_assessment_question_multiple_response_option, :correct, question: subject)
+        end
+
+        it 'returns multiple response' do
+          expect(subject.type).to eq I18n.t('course.assessment.question.multiple_responses.type.multiple_response')
+        end
+      end
+
+      context 'when question has one correct option' do
+        subject { build(:course_assessment_question_multiple_response) }
```

Comment:
To solve https://github.com/Coursemology/coursemology2/pull/2858#discussion_r172407739, you need to add in the factory trait here, so `build(:course_assessment_question_multiple_response, :multiple_choice)`. You can see more from [`spec/factories/course_assessment_question_multiple_responses.rb`](https://github.com/Coursemology/coursemology2/blob/master/spec/factories/course_assessment_question_multiple_responses.rb#L33-L35).